setup.py

```diff
@@ -30,7 +30,7 @@
         "ncempy >= 1.8.1",
         "matplotlib >= 3.2.2",
         "scikit-image >= 0.17.2",
-        "scikit-learn >= 0.23.2",
+        "scikit-learn >= 0.23.2, < 1.5",
```

Comment:
looks good for now, but let's loop back and relax this once they've figured out their regression